packages/expo-updates/android/src/main/java/expo/modules/updates/manifest/BareManifest.java

```diff
@@ -86,7 +86,7 @@ public UpdateEntity getUpdateEntity() {
   public ArrayList<AssetEntity> getAssetEntityList() {
     ArrayList<AssetEntity> assetList = new ArrayList<>();
 
-    AssetEntity bundleAssetEntity = new AssetEntity(null, "js");
+    AssetEntity bundleAssetEntity = new AssetEntity("bundle-" + mCommitTime.getTime(), "js");
```

Comment:
If I understand correctly, the server needs to generate this exact same key for the `bundleKey` property. If that's right, we'll want to make it easy to implement on the server and identical across Android and iOS so we don't need platform-specific logic. I'm a bit worried that working with the timestamps will be difficult because on Android, the timestamp gets converted into a Date object and then back into milliseconds.

Do you agree with this? Would using the manifest's `id` field instead of the timestamp be easier?

Reply:
This is only for no-publish manifests created when making bare workflow builds, so the server shouldn't need to worry about this at all -- the important thing is just that it's distinct from any other bundle. But yeah, the `id` seems just as unique and maybe more predictable, so there's no downside to switching to that.